cl/scrapers/management/commands/clone_from_cl.py

```diff
@@ -0,0 +1,298 @@
+docker-compose -f docker/courtlistener/docker-compose.yml exec cl-django python manage.py clone_from_cl --type Docket --id 5377675
+docker-compose -f docker/courtlistener/docker-compose.yml exec cl-django python manage.py clone_from_cl --type Person --id 16207
+docker-compose -f docker/courtlistener/docker-compose.yml exec cl-django python manage.py clone_from_cl --type Court --id usnmcmilrev
+
+
+This tool is only for development purposes, to enable it you need to set
+environment variable DEVELOPMENT to True. Also need to set CL_API_TOKEN env
+variable.
+
+This is still work in progress, some data is not cloned yet.
+
+"""
+
+import os
+import sys
+
+import environ
+import requests
+from django.db import transaction
+from django.utils.dateparse import parse_date
+
+from cl.lib.command_utils import VerboseCommand
+from cl.people_db.models import Person
+from cl.search.models import Citation, Court, Docket, Opinion, OpinionCluster
+from cl.search.tasks import add_items_to_solr
+
+env = environ.FileAwareEnv()
+DEVELOPMENT = env.bool("DEVELOPMENT", default=False)
+
+VALID_TYPES = ("Opinion", "Docket", "Person", "Court")
+
+cluster_endpoint = "https://www.courtlistener.com/api/rest/v3/clusters/"
+people_endpoint = "https://www.courtlistener.com/api/rest/v3/people/"
+courts_endpoint = "https://www.courtlistener.com/api/rest/v3/courts/"
+
+
+class Command(VerboseCommand):
+    help = (
+        "A helper function clone a data from courtlistener.com into local "
+        "environment "
+    )
+
+    def __init__(self, *args, **kwargs):
+        super(Command, self).__init__(*args, **kwargs)
+        self.verbosity = None
+        self.options = []
+        self.type = None
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--type",
+            type=str,
+            choices=VALID_TYPES,
+            help="Object type to clone. Current choices are %s"
+            % ", ".join(VALID_TYPES),
+        )
+
+        parser.add_argument(
+            "--id",
+            help="Object id, " "object id to clone",
+        )
+
+    def handle(self, *args, **options):
+        super(Command, self).handle(*args, **options)
+        self.verbosity = int(options.get("verbosity", 1))
+        self.type = options.get("type")
+        self.id = options.get("id")
+
+        if not self.id:
+            self.stdout.write("Object id required!")
+            sys.exit(1)
+
+        if DEVELOPMENT:
+            if self.type == "Opinion":
+                self.clone_opinion()
+            elif self.type == "Docket":
+                self.clone_docket()
+            elif self.type == "Person":
+                self.clone_person()
+            elif self.type == "Court":
+                self.clone_court()
+            else:
+                self.stdout.write("Invalid type!")
+
+        else:
+            self.stdout.write("Command not enabled for production environment")
```

Comment:
A couple things here:
 - Better to do an early abort `if not DEVELOPMENT` than to do an if/else with nested if statements.
 - This is a good time to use `match`.
 - I tweaked this to use the full model name (as mentioned earlier)
 - This moves the methods on the class to functions on the module by removing `self`. More on this in a sec.

```suggestion
        if not DEVELOPMENT:
            self.stdout.write("Command not enabled for production environment")
        
        match self.type:
            case "search.Opinion":
                clone_opinion()
            case "search.Docket":
                clone_docket()
            case "person_db.Person":
                clone_person()
            case "search.Court":
                clone_court()
            case _:
                self.stdout.write("Invalid type!")
```